### ISSUE TITLE
Integration to mathcomp

### ISSUE THREAD

**Issue by CohenCyril** [thery/grobner#1]
Hi @thery how about integrating your development to mathcomp?

**Comment by thery** [comment]
:+1: but what about multinomial is it still in?

**Comment by palmskog** [comment]
@thery how about adding `grobner.v` to CoqEAL? This way, more people will be able to use it, since CoqEAL is part of the Coq Platform. CoqEAL already depends on Multinomials. Finally, the CoqEAL theory is already a form of incubator for results to add to regular MathComp.

**Comment by thery** [comment]
@palmskog Yep that was the plan a long time ago (beginning of the phD of Damien (Roughling)), I see if I can find some energy to properly do this

**Comment by thery** [comment]
@palmskog https://github.com/coq-community/coqeal/pull/70
